codebuild_specs/e2e_workflow.yml

```diff
@@ -131,33 +131,34 @@ batch:
           CLI_REGION: eu-central-1
       depend-on:
         - publish_to_local_registry
-    - identifier: containers_api_secrets_schema_function_1_api_3_api_1
```

Comment:
Why are we changing some of these identifiers and regions?

Reply:
That's because of the split-e2e pre-commit hook we have.